### ISSUE TITLE
Feature request: add stability to commands and options

### ISSUE THREAD

**Issue by bitliner** [yargs/yargs#1638]
**Context**

Sometimes a command could be not stable yet.

Same for command options of a stable command.

A similar scenario is NodeJS standard API. Node JS manages 3 levels of stability (see [here](https://nodejs.org/api/documentation.html#documentation_stability_index) )

- stability 0 - deprecated
- stability 1 - Experimental (eg Use of the feature is not recommended in production environments)
- stability 2 - Stable

**Problem**

Yargs does not provide a way to set which options/commands are experimental or stable.

**Question**

Do you plan to add support for it?

**Comment by mleguen** [comment]
@bcoe Couldn't that be an evolution of the recently added deprecated feature?

**Comment by bcoe** [comment]
@mleguen yeah, I think that's a good idea. @laggingreflex thoughts on this one?

**Comment by laggingreflex** [comment]
Yep, I like this too. This does seem to be a direct evolution.

We can change `deprecated:bool|string` API to `stability:string`?

**Comment by jgoux** [comment]
This feature would be great, I'm writing a tool to automatically generate documentation out of a yargs CLI program and we have experimental commands (I have to edit the doc afterward manually currently).

**Comment by shadowspawn** [comment]
This issue has not had any activity in over six months. It isn't likely to get acted on due to this report.

Feel free to open a new issue if it comes up again, with new information and renewed interest.
 
Thank you for your contributions.
